### DISCUSSION TITLE
quarto publish errors 422 and 429 (`quarto publish` from RStudio)

### DISCUSSION THREAD

**Discussion by pbosetti** [quarto-dev/quarto-cli#6652] [Q&A]
### Description

Today I have started experiencing error when trying to publish on quarto.pub. I can still update existing projects with the command `quarto publish .`, but when I create a new `qmd` presentation in RStudio and then issue the command `quarto publish .` it asks for the service and the account to use, the name of the presentation then immediately crashes with js errors. 

At first I was getting a `ERROR: API Error: 422 - Unprocessable Entity`, then it started to give `ERROR: API Error: 429 - Too Many Requests`. At the same time, I still can publish existing documents with no problems, but I cannot publish any new presentation/document.

Any idea?

I also would like to reset the account info stored by the quarto command, but I can't find the file where those info are saved. Anyone knowing where that file is on MacOS?

Thanks,
Paolo

**Comment by mcanouil** [comment]
See:
- https://github.com/quarto-dev/quarto-cli/issues/6624

**Reply by pbosetti** [reply]
I saw that, but it's not of much help. Additionally, I am also asking where quarto is storing account information, which is not answered there nor elsewhere (nowhere easy to find, at least).

**Reply by mcanouil** [reply]
> I saw that, but it's not of much help.

Indeed, because the issue is not user side. So, there is nothing you can do.

> Additionally, I am also asking where quarto is storing account information, which is not answered there nor elsewhere (nowhere easy to find, at least).

I saw that but I can't give an answer I do not have.

**Reply by mcanouil** [reply]
From the codebase, you can look for something `HOME/.../quarto/publish/accounts/quartopub/accounts.json`

**Reply by pbosetti** [reply]
That is very helpful, thanks!

**Reply by pbosetti** [reply]
Hey @mcanouil just out of curiosity: can you publish a new, plain presentation to quarto.pub? I still get the same error on different computers, so it's either my account or a server issue. If you can publish, then it's an account issue... Thanks in advance!

**Reply by mcanouil** [reply]
There was another issue regarding publishing new website which had the exact same root cause.
It should be resolved now.
